bot/exts/help_channels/_message.py

```diff
@@ -124,54 +123,59 @@ async def dm_on_open(message: discord.Message) -> None:
         )
 
 
-async def notify(channel: discord.TextChannel, last_notification: t.Optional[Arrow]) -> t.Optional[Arrow]:
+async def notify_none_remaining(channel: discord.TextChannel) -> None:
```

Comment:
Because `bot.instance` is available, this argument can be omitted. Instead, the function body can be hard-coded to use `constants.HelpChannels.notify_channel` as the channel. I don't see any benefit to allowing arbitrary channels in this case. Thoughts?

Reply:
I agree, the notify_channel is already configurable via the `config.yml`